README.md

```diff
@@ -173,6 +177,8 @@ date: 2016-01-21 18:45:55
 ---
 ```
 
+Please note that you must specify `title` attribute using the same name with the folder, for example, if we add `blog_root/source/tags/index.md`, the `title` attribute in `index.md` must be `tags`.
```

Comment:
no need this paragraph. It is not that title must be the same with filename, but the title must be exactly `tags` or `categories` for rendering corresponding pages